cmd/savepcdfiles/main.go

```diff
@@ -30,7 +31,8 @@ func main() {
 
 // Arguments for the command.
 type Arguments struct {
-	Port       utils.NetPortFlag `flag:"0"`
+	TimeDelta  int               `flag:"0"`
```

Comment:
New arg time delta (see below)

Reply:
[nit] same as above, can you make the time unit clear?

I love that we can specify the delta in the command line now!

Reply:
Changed to Milliseconds (Millis has been used before to specify millimeters)

Reply:
Thank you :)